### PR TITLE
Uk/mobile navbar menu

### PR DESCRIPTION
**Pull request by orenku**

### CODE REVIEW

File: src/components/Layout.tsx [hunk 1]
```diff
@@ -42,19 +55,54 @@ const Layout: FC<PropsWithChildren> = ({ children }) => {
...
+                enter="transition ease-out duration-100"
+                enterFrom="transform opacity-0 scale-95"
+                enterTo="transform opacity-100 scale-100"
+                leave="transition ease-in duration-75"
+                leaveFrom="transform opacity-100 scale-100"
+                leaveTo="transform opacity-0 scale-95"
+              >
+                <Menu.Items className="absolute right-0 my-3 w-56 origin-top-right divide-y divide-gray-100 bg-white text-xl text-black focus:outline-none">
+                  {menuItems.map((item, index) => (
+                    <Menu.Item key={`menu-item-${index}`}>
```
Comment (vanillaHill) [inline src/components/Layout.tsx:94]: I think you don't need to add key here too

File: src/components/Layout.tsx [hunk 2]
```diff
@@ -42,19 +55,54 @@ const Layout: FC<PropsWithChildren> = ({ children }) => {
...
               <nav className="hidden space-x-10 lg:block" aria-label="Global">
-                <Link className="hover:text-pink-400" href="/vaults">
-                  Vaults
-                </Link>
-                <Link className="hover:text-pink-400" href="/lend">
-                  Lend
-                </Link>
+                {menuItems.map((item, index) => (
+                  <Link
+                    key={`menu-item-${index}`}
```
Comment (vanillaHill) [inline src/components/Layout.tsx:60]: I think you don't need to add key here

**Review by vanillaHill — CHANGES_REQUESTED** [review]

File: src/components/Layout.tsx [hunk 3]
```diff
@@ -1,7 +1,9 @@
+import { Menu, Transition } from "@headlessui/react"
 import dynamic from "next/dynamic"
 import Image from "next/image"
 import Link from "next/link"
-import { FC, PropsWithChildren } from "react"
+import { FC, Fragment, PropsWithChildren } from "react"
+import { RxHamburgerMenu } from "react-icons/Rx"
```
Comment (aquaticone) [inline src/components/Layout.tsx:6]: Lowercase rx and the build will pass

File: src/components/Layout.tsx (hunk 2 above)
Comment (aquaticone) [reply src/components/Layout.tsx:60]: Key is needed because it's inside a map

**Review by aquaticone — CHANGES_REQUESTED** [review]
Would like to give it a test once the preview build is working, but looks good.

File: src/components/Layout.tsx (hunk 2 above)
Comment (orenku) [reply src/components/Layout.tsx:60]: Key is needed because it's inside a map

File: src/components/Layout.tsx (hunk 3 above)
Comment (orenku) [reply src/components/Layout.tsx]: Fixed

File: src/components/NetworkSelector.tsx [hunk 4]
```diff
@@ -38,12 +38,12 @@ const NetworkSelector: FC<NetworkSelectorProps> = () => {
...
 
   return (
-    <div className="min-w-30 mr-3 text-right">
+    <div className="min-w-30 mr-2 text-right md:mr-3">
       <Menu as="div" className="relative inline-block text-left">
         <div>
           <Menu.Button
             className={clsxm(
-              "text-medium inline-flex w-full items-center justify-center space-x-2 rounded-md border-2 bg-black/20 px-4 py-3 font-medium text-white hover:bg-black/30",
+              "text-medium x x inline-flex w-full items-center justify-center space-x-2 rounded-md border-2 bg-black/20 py-1 px-2 text-sm font-medium text-white hover:bg-black/30 md:py-3 md:px-4 md:text-base",
```
Comment (aquaticone) [inline src/components/NetworkSelector.tsx:46]: Remove `x x` from here

**Review by aquaticone — CHANGES_REQUESTED** [review]

File: src/components/NetworkSelector.tsx (hunk 4 above)
Comment (orenku) [reply src/components/NetworkSelector.tsx]: Where this came from !@?

removed

**Review by aquaticone — APPROVED** [review]

**Pull request merged**